### DISCUSSION TITLE
JSX element class does not support attributes because it does not have a 'props' property.

### DISCUSSION THREAD

**Discussion by chiester** [vazco/uniforms#1027] [Q&A] [answered]
Any idea why I would get this error on `<AutoForm>`? 

In the console: 

```
Type error: No overload matches this call.
  The last overload gave the following error.
    Type '{ children: Element[]; schema: JSONSchemaBridge; model: any; onSubmit: any; onValidate: (model: BaseCase, error: any) => Promise<any>; ref: (ref: any) => any; onChangeModel: (model: BaseCase) => void; style: { ...; }; }' is not assignable to type 'IntrinsicAttributes & IntrinsicClassAttributes<BaseForm<unknown, BaseFormProps<unknown>, BaseFormState<unknown>>> & Pick<...> & Partial<...> & Partial<...>'.
      Property 'onChangeModel' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<BaseForm<unknown, BaseFormProps<unknown>, BaseFormState<unknown>>> & Pick<...> & Partial<...> & Partial<...>'.

  244 |       onValidate={onValidate}
  245 |       ref={(ref: any) => (formRef = ref)}
> 246 |       onChangeModel={(model: BaseCase) => {
      |       ^
  247 |         setFormModel(model)
  248 |       }}
```

**Comment by radekmie** [comment]
Hi @chiester. Could you include the versions of all uniforms packages you're using? We've recently improved typings of all form components (#1003) and maybe something went sideways.

**Comment by chiester** [comment, marked as answer]
"uniforms": "^3.5.5",
    "uniforms-bridge-json-schema": "^3.5.5",
    "uniforms-material": "^3.5.5"

I tried the workaround in the FAQ and that resolved the warning.

**Reply by wadamek65** [reply]
Glad you resolved it, but if I may ask could you please post the code with the usage that resulted in this error and the full error you were getting? We would like to confirm whether this is something that went wrong on our side and we need to adjust it or not.

**Reply by chiester** [reply]
It's a proprietary product in development, so I can't publicly post the code. I installed 3.6.0 and still get the same warning on the `onChangeModel` attribute of AutoForm without your workaround.
